Assets/Scripts/Spawner.cs

```diff
@@ -0,0 +1,23 @@
+﻿using UnityEngine;
```

Comment:
Not quite sure what this class does, maybe it could use a better name since it does not seem to spawn anything